src/compiler/diagnosticMessages.json

```
@@ -659,7 +659,7 @@
         "category": "Error",
         "code": 1207
     },
-    "All files must be modules when the '--isolatedModules' flag is provided.": {
+    "'{0}' cannot be compiled under '--isolatedModules' because it is considered a global script file. Ensure that it contains an import, export, or an empty 'export {}' statement to make it a module.": {

```

Assuming that this error happens only on non-modules, it can be simplified:

```suggestion
    "'{0}' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.": {
```

---

@sandersn Thanks